test/e2e/performanceprofile/functests/8_performance_workloadhints/workloadhints.go

```diff
@@ -149,11 +158,11 @@ var _ = Describe("[rfe_id:49062][workloadHints] Telco friendly workload specific
 				By("Updating the performance profile")
 				profiles.UpdateWithRetry(profile)
 
-				By("Applying changes in performance profile and waiting until mcp will start updating")
-				mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdating, corev1.ConditionTrue)
+				testlog.Infof("Applying changes in performance profile and waiting until %s will start updating", resourcePool)
+				profilesupdate.WaitForTuningUpdating(ctx, profile)
 
-				By("Waiting for MCP being updated")
-				mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdated, corev1.ConditionTrue)
+				testlog.Infof("Waiting when %s finishes updates", resourcePool)
+				profilesupdate.WaitForTuningUpdated(ctx, profile)
```

Comment:
please replace all `profilesupdate.WaitForTuningUpdated` occurrences with `profilesupdate.PostUpdateSync`

Reply:
fixed in the latest commit